index.html

```diff
@@ -0,0 +1,459 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars" id="menu-button">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="desktop-nav" class="desk-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="menu hide" id="menu-id">
+      <button id="close">
+        <i>X</i>
+      </button>
```

Comment:
- [ ] We understand you have spent much time working on this "Mobile menu" project 😄 . however, you should make sure the `x`  button is working because it will increase the user experience and also it's part of the requirement. 👍 

![mo](https://user-images.githubusercontent.com/57408419/218091804-5f74ede1-fb93-4aeb-870a-2573101820ff.jpg)